project/db/migrate/20230117045308_add_occupation_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOccupationToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :occupation, :string
```

Comment:
@y-kohei1228 
[imo]
NOT NULL 成約を入れると `''` と `NULL` の扱いに困らなくてよいです

```suggestion
    add_column :users, :occupation, :string, null: false, default: ""
```